src/App.css

```diff
@@ -79,3 +79,29 @@ body {
   width: 24px;
   height: 24px;
 }
+
+.footer {
+  position:fixed;
+  left:0px;
+  bottom:0px;
+  height:50px;
+  width:100%;
+  background-color: rgba(255, 255, 255, .85);
+  z-index: 2;
+}
+
+.footer .left {
+    float: left;
+}
+
+.footer .right {
+    float: right;
+}
+
```

Comment:
In fact, I would like to display name of layouts (for example: bedroom, garage, etc.) in the footer.
But you're right to avoir overlaping of bottom widgets I added a 50px margin.